.golangci-extra.yml

```diff
@@ -20,3 +20,4 @@ linters:
       checks:
         - all
         - -QF1008 # https://staticcheck.dev/docs/checks/#QF1008 Omit embedded fields from selector expression.
+        - -ST1000 # https://staticcheck.dev/docs/checks/#ST1000 Incorrect or missing package comment.
```

Comment:
I don't like it. We specifically added it so the "new code" linter is more restrictive.

I know that sometimes "new code" is actually old code with some changes, but there's nothing we can do about it.

So, please drop this commit; we DO want the linter to fail if "new code" has no package comment (and we can force-merge PRs, if needed).

Reply:
I do not mind, I can add the package comment to avoid force push